cfme/tests/containers/test_tags.py

```diff
@@ -44,37 +50,67 @@ def get_collection_entity(appliance, collection_name, provider):
         return item_collection.instantiate(**d)
 
 
+def verify_tags(obj_under_test, tag, details, dashboard, soft_assert):
+
+    obj_under_test.add_tag(tag=tag, details=details, dashboard=dashboard)
+
+    tags = obj_under_test.get_tags()
+
+    assert any(
+        object_tags.category.display_name == tag.category.display_name and
+        object_tags.display_name == tag.display_name for object_tags in tags), (
+        "{}: {} not in ({})".format(tag.category.display_name, tag.display_name, str(tags)))
+
+    obj_under_test.remove_tag(tag=tag, details=details)
+
+    post_remove_tags = obj_under_test.get_tags()
+    if post_remove_tags:
+        for post_tags in post_remove_tags:
+            soft_assert(
+                not post_tags.category.display_name == tag.category.display_name and
+                not post_tags.display_name == tag.display_name)
+
+
 @pytest.mark.parametrize('test_param', container_test_items, ids=[cti for cti in
                                                                   container_test_items])
 @pytest.mark.parametrize('tag_place', [True, False], ids=['details', 'list'])
-def test_tag_container_objects(soft_assert, test_param, appliance, provider, request, tag,
+def test_tag_container_objects(soft_assert, test_param, appliance, provider, tag,
                                tag_place):
     """ Test for container items tagging action from list and details pages
 
     Polarion:
         assignee: juwatts
-        caseimportance: medium
         casecomponent: Containers
+        caseimportance: medium
         initialEstimate: 1/6h
     """
 
     obj_under_test = get_collection_entity(appliance=appliance, collection_name=test_param,
                                            provider=provider)
 
-    obj_under_test.add_tag(tag=tag, details=tag_place)
+    verify_tags(obj_under_test=obj_under_test, tag=tag, details=tag_place, dashboard=False,
+                soft_assert=soft_assert)
 
-    tags = obj_under_test.get_tags()
 
-    assert any(
-        object_tags.category.display_name == tag.category.display_name and
-        object_tags.display_name == tag.display_name for object_tags in tags), (
-        "{}: {} not in ({})".format(tag.category.display_name, tag.display_name, str(tags)))
+@pytest.mark.parametrize('test_param', bz_1665284_test_items, ids=[bti for bti in
```

Comment:
Required:
No need to add `ids`; If you are not manipulating ids with different tags. You are using same names.